app/javascript/components/pages/FateNew.js

```diff
@@ -29,8 +29,18 @@ class FateNew extends Component {
   };
   handleSubmit = (e) => {
     e.preventDefault();
-    this.props.createNewEvent(this.state.form);
-    this.setState({ submitted: true });
+    const { description, url, category, is_public } = this.state.form;
+    if (
+      (!description && description === "") ||
```

Comment:
These can be simplified to just `!description`, etc...

Reply:
you actually cant it does not work unless both are used in conjunction, i have no idea why tried both of them separately and they only work together